Makefile

```diff
@@ -94,6 +96,11 @@ minikube_iso:
 	$(MAKE) BR2_EXTERNAL=../../deploy/iso/minikube-iso minikube_defconfig -C $(BUILD_DIR)/buildroot
 	$(MAKE) -C $(BUILD_DIR)/buildroot
 	mv $(BUILD_DIR)/buildroot/output/images/rootfs.iso9660 $(BUILD_DIR)/minikube.iso
+else
+	docker run --env IN_DOCKER=1 --workdir /mnt --entrypoint /usr/bin/make \
```

Comment:
I tried but this will call the build of the image inside the docker container and fail 

-> outside of the container call to buildroot-image as dependency
-> once this is done run the docker run command
-> docker run call the same target triggerring the buildroot dependency again
-> fail
We can create an indirection target, like 
```
out/minikube.iso: buildroot-image
    docker run ...

_out/minikube.iso:
    git clone buildroot ....
```

But was not sure of this, let me know